2013/data_file_specification.json

```diff
@@ -260,6 +390,9 @@
                "end": "54",
                "length": "5",
                "dataType": "N",
+               "format": "currency,thousands",
```

Comment:
To add to @LinuxBozo's comment above, I considered the array, but if we decide to go with a non-string value for the format, then I'd opt to make the value an object rather than array so that we could more easily pull the format values by a key rather than their index in the array.